CHANGELOG.md

```diff
@@ -8,6 +8,13 @@ and this project adheres to
 
 ## [Unreleased]
 
+## [1.2.0] - 2023-08-28
+
+### Changed
+
+- Add languages to the `CourseRun` serializer
+- Rename certificate field to certification_definition into the `ProductSerializer`
```

Comment:
I think it is : Rename certificate field to `certificate_definition` into the `ProductSerializer`

Reply:
You're right !